xarray/core/options.py

```diff
@@ -18,6 +44,9 @@ class set_options(object):
       Default: ``80``.
     - ``arithmetic_join``: DataArray/Dataset alignment in binary operations.
       Default: ``'inner'``.
+    - ``io_engines``: List of backend data I/O engines. Default: All the
```

Comment:
should add something like "to try in order when saving/loading data without an explicit engine"